tools/install-tcmalloc.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+#
+# Script to install libtcmalloc.so in the current WiredTiger git
+# repository.
+#
+# This is serves the same purpose, but is NOT THE SAME as:
+#
+# test/evergreen/tcmalloc_install_or_build.sh
+#
+# and the tcmalloc source version referenced here should be updated in
+# sync with that script.
+
+set -euf -o pipefail
+
+die() { echo "$@"; exit 1; }
+
+HERE=$(git rev-parse --show-toplevel) || die "FATAL Run in root of git workspace"
+[[ -f ${HERE}/CMakeLists.txt ]] || die "Does not look like a WT workspace - no CMakeLists.txt"
+[[ -d ${HERE}/TCMALLOC_LIB ]] && die "TCMALLOC_LIB already present"
```

Comment:
This probably should just `exit 0` as the purpose of the script is to make tcmalloc available. If it's already there, nothing to do then.

Reply:
It's a fair point, but this is a developer run script, and the developer thought they need to build tcmalloc: and the script is no repurposed to build tcmalloc only and can't so it's still arguably an error, and not likely to upset the developer.

Reply:
Given the test is simplistic in its test I think it is best to warn the developer nothing was done.